etc/README.md

```diff
@@ -0,0 +1,46 @@
+# Installation instructions
+
+## bash
+
+Open your `.bashrc` file and add:
+
+> If you want to set up Git to automatically have Bash shell completion for all users, copy the `hub.bash_completion` script to the `/opt/local/etc/bash_completion.d` directory on Mac systems or to the `/etc/bash_completion.d/` directory on Linux systems. ([Source](https://git-scm.com/book/en/v1/Git-Basics-Tips-and-Tricks#Auto-Completion))
+
+* [Link to git auto-completion bash file](https://github.com/git/git/blob/master/contrib/completion/git-completion.bash)
+
+```sh
+# Make sure you've aliased hub to git
+eval "$(hub alias -s)"
+
+# And make sure that the git auto-completion is being loaded
+if [ -f /path/to/git-completion.bash ]; then
+    . /path/to/git-completion.bash
```

Comment:
This is not necessary. Let's assume they're already successfully loading the git completion script, and they just want to extend the functionality with hub.